### PR TITLE
py-rbtools: update to 1.0.3, (+ add subports)

### PR DESCRIPTION
**Pull request by iEFdev**
#### Description

  - Update to version: 1.0.3
  - In order to have duplicity to use python38, this port
    needs a py38 subport.

###### Type(s)

- [x] update
- [x] enhancement

###### Tested on

macOS 10.7.5 11G63
Xcode 4.6.3 4H1503

###### Verification

Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port install`?

###### Notes

To PR is a part of the `duplicity` update.

### CODE REVIEW

File: python/py-rbtools/Portfile [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
-supported_archs noarch
-license         MIT
-description     Tools for Review Board integration
-
-long_description \
-    Tools for integrating with Review Board, a powerful web-based code review \
-    tool that offers developers an easy way to handle code reviews. \
-    This package provides \"post-review\", a command-line tool that simplifies \
-    both creating and updating review requests.
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```
Comment (ryandesign) [inline python/py-rbtools/Portfile:1]: Please use the standard modeline.

Please don't completely change the whitespace of the port *and* make functional changes all at once. Do the whitespace adjustment and the addition of the modeline in one commit, then make a second commit with the functional changes.

Comment (iEFdev) [reply python/py-rbtools/Portfile]: Ok. Yes, smaller/separate commits are better. Didn't know we could do more than one commit _(as in “… squashed and minimized your commits?”)_.

Comment (ryandesign) [reply python/py-rbtools/Portfile]: > Didn't know we could do more than one commit _(as in “… squashed and minimized your commits?”)_.

We ask you to make sure you've "squashed and minimized your commits", meaning you've squashed any series of work-in-progress changes into the minimum number of logically distinct commits to accomplish your goal.

You've separated it into two commits now, which is good, but the commit messages could be improved. I suggest the first commit message be changed to:

```
py-rbtools: Add modeline and adjust whitespace
```

and the second commit message be changed to:

```
py-rbtools: Update to 1.0.3 and add py3* subports

In order to have duplicity to use python38, this port needs a py38
subport.
```

Comment (iEFdev) [reply python/py-rbtools/Portfile]: > … into the minimum number of logically distinct commits to accomplish your goal.

Thanks for clarifying that. :+1:

Yes, I was thinking about that _after_ the push, that I maybe shoul've changed the msg's a little more than I did.

**Pull request merged**